chain/rpc/Cargo.toml

```diff
@@ -14,7 +14,7 @@ crate-type = ["rlib"]
 [features]
 default = []
 prometheus = ["dep:hopr-metrics"]
-runtime-tokio = ["hopr-async-runtime/runtime-tokio"]
+runtime-tokio = ["hopr-async-runtime/runtime-tokio", "reqwest"]
```

Comment:
We should use the `dep:` prefix to distinguish from features and dependencies.

So it should be `dep:reqwest`, like it is with `hopr-metrics` on the line above.

Reply:
```suggestion
runtime-tokio = ["hopr-async-runtime/runtime-tokio", "dep:reqwest"]
```

Are the features not needed anymore?

Reply:
You mean different runtimes?

Reply:
@NumberFour8 meant the modification I did as a suggestion.

Reply:
That I've updated in the code. I was asking what you meant by "Are the features not needed anymore?"